# Define the goal
action_server/Point2D[] waypoints
int32 mode
action_server/Point2D tag_point

---
# Define the result
bool state
geometry_msgs/Point odom
---
# Define a feedback message
float32 dist
